inventory/environ.py

```diff
@@ -350,40 +342,61 @@ def merge_dict(dict1, dict2, path=None):
             dict1[key] = dict2[key]
     return dict1
 
-def mergeDefaultSplunkVariables(vars_scope, url):
-    url = url.strip()
-    if not url or len(url) == 0:
+def mergeDefaults(vars_scope, src):
```

Comment:
Broke this function down into `mergeDefaultsFromURL()` and `mergeDefaultsFromFile()` while this function itself does the strategy handling.